Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -762,6 +784,25 @@ DayOfWeek: !dictionary
     zo.: 0
     zon: 0
     zon.: 0
+    monday: 1
```

Comment:
@LionbridgeCSII why is this necessary? Are there situations in Dutch which require mixed English + Dutch recognition?

Reply:
this is necessary because with certain patterns the extractor extracts the English version of the weekday and then looks for it in DayOfWeek dictionary. So, we included it because it would fail otherwise. (Note that the same has been done for Italian and Turkish)

Reply:
This is probably something that should be refactored in the future. But it was marked as low priority to change.